doc/strategies/L2Deposit.md

```diff
@@ -0,0 +1,23 @@
+# L2Deposit
+
+The `L2Deposit` is a utility contract that helps users to bridge their assets from L1 to supported L2. Users can onboard different L2 without approving tokens to each L2 gateway contract separately.
+
+Currently `L2Deposit` supports following L2 networks:
+
+-   Arbitrum
+-   Optimism
+
+## Parameters
+
+| Field              |  Type   | Description                                                                               |
+| ------------------ | :-----: | ----------------------------------------------------------------------------------------- |
+| L2Identifier       |  enum   | The identifier of L2 network.                                                             |
+| l1TokenAddr        | address | The address of token on L1.                                                               |
+| l2TokenAddr        | address | The address of token on L2.                                                               |
+| sender             | address | The address of sender on L1.                                                              |
+| recipient          | address | The address of recipient on L2.                                                           |
+| arbitrumRefundAddr | address | The address for arbitrum refund (if the eventual gas cost is less than the pre-paid gas). |
```

Comment:
[just murmur] 

To my surprise I didn't notice before that `arbitrumRefundAddr` field is added into the deposit param struct 😱 

I think it would be better to put in bridge specific data. But this doesn't affect the functionality, just skip my murmur at this moment. 🤣

Reply:
Yeah agree it's better to put it in bridge specific data. Maybe open a ticket for it and leave it as is for now?

Reply:
Ticket opened. Will fix in another PR.